packages/paste-website/src/pages/page-templates/object-details/index.mdx

```diff
@@ -0,0 +1,142 @@
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Object details');
+  return {
+    props: {
+      data: {
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<content>
+
+<NormalizedPatternHeader
+  description={meta.description}
+  name={meta.title}
+  data={props.data}
+  categoryRoute={SidebarCategoryRoutes.PAGE_TEMPLATES}
+/>
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+## About Object details
+
+Object details give customers a primarily read-only view of the properties of an object, such as a message log or sent email.
+
+If the customer is primarily editing information, use the [Settings](/page-templates/settings) page instead.
+
+## Accessibility
+
+A well-structured [document hierarchy](https://www.w3.org/WAI/tutorials/page-structure/) helps provide efficient in-page navigation for keyboard users, assistive technologies, and mobile web users.
+
+Proper hierarchy allows users to skip directly to content that is most relevant to them. This is especially important on an Object details page where there may be many different types of information and customers are more likely to skim through.
+
+To structure a page:
+
+- Use [Headings](/components/heading) to organize the page.
+- Use Headings in sequential order. For example, don’t place an H4 directly after an H2.
+- Use sentence case for headers (“Message date” not “Message Date”).
+- Use a [Description List](/components/description-list) to define relationships between a property and its label.
+- Use [Tables](/components/table) to organize data that are meant to be compared.
+
+## Ingredients
+
+<Ingredients>
+  <Required>
+    <UnorderedList>
+      <ListItem>
+        <Anchor href="/components/breadcrumb">Breadcrumb</Anchor> and{' '}
+        <Anchor href="/components/heading">Heading</Anchor>
+      </ListItem>
+      <ListItem>
+        <Anchor href="/components/description-list">Description List</Anchor>
+      </ListItem>
+      <ListItem>
+        <Anchor href="/components/stack">Stack</Anchor> and <Anchor href="/components/grid">Grid</Anchor>
+      </ListItem>
+    </UnorderedList>
+  </Required>
+  <Related>
+    <UnorderedList>
+      <ListItem>
+        <Anchor href="/components/anchor">Anchor</Anchor>
+      </ListItem>
+      <ListItem>
+        <Anchor href="/components/button">Button</Anchor> and{' '}
+        <Anchor href="components/button-group">Button Group</Anchor>
+      </ListItem>
+      <ListItem>
+        <Anchor href="/components/in-page-navigation">In Page Navigation</Anchor> and{' '}
+        <Anchor href="/components/tabs">Tabs</Anchor>
+      </ListItem>
+      <ListItem>
+        <Anchor href="/components/icon">Icon: Information</Anchor>
+      </ListItem>
+    </UnorderedList>
+  </Related>
+</Ingredients>
```

Comment:
suggestion(non-blocking): I'd still like to see single imports rather than composed JSX in MDX files. I suspect we're going to have to replace it all for the MDX2 upgrade